content/news/012.dd-ix-at-ripe89.en.md

```diff
@@ -0,0 +1,17 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk and Memorable Experiences
+title: DD-IX at RIPE89 in Prague
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens, and Melody from DD-IX visited the RIPE community at this year's autumn RIPE event in Prague
+RIPE event in Prague. It was a great week with lots of fun and interesting conversations about the future of 
```

Comment:
It was a great week with interesting discussions about the future of the Internet in Europe. Our personal highlight was the lightning talk given by Marcel ;). He presented our RIPE Labs Serie "IXP-from-Scratch" to more than 600 participants. Haven't had the chance to listen to his presentation yet? A recorded version is available on the [RIPE89 web page](https://ripe89.ripe.net/archives/video/1542/).